react-native/react/tracker/proofs.render.desktop.js

```diff
@@ -2,18 +2,18 @@
 
 import React, {Component} from '../base-react'
 import commonStyles, {colors} from '../styles/common'
-
 import {checking, revoked} from '../constants/tracker'
 import {metaNew, metaUpgraded} from '../constants/tracker'
+import {remote} from 'electron'
```

Comment:
i had problems with that not working before with babel, maybe they fixed it... also the other branch changed this code so it won't really work anymore. it's now
```
const shell = electron.shell || electron.remote.shell
```

i really REALLY hate that electron exposes different apis like this